gluten-core/src/main/scala/io/glutenproject/expression/WindowFunctionsBuilder.scala

```diff
@@ -21,11 +21,14 @@ import io.glutenproject.substrait.expression.ExpressionBuilder
 
 import org.apache.spark.sql.catalyst.expressions.{Expression, WindowExpression, WindowFunction}
 
+import java.lang.{Long => JLong}
+import java.util.{HashMap => JHashMap}
```

Comment:
Can this import statement generated by some kind of tools?

Reply:
I made the modifications manually. However, upon carefully examining the code of Spark, I noticed that it actually has a lot of inconsistency in its usage. There are various conventions being used, with most of them using `JList`, but `java.util.List` and `util.List` are still present as well. I am now questioning whether it is necessary to make these modifications here, as it would be difficult to ensure that all future developers maintain a consistent style.

Reply:
> as it would be difficult to ensure that all future developers maintain a consistent style.

Yes so it's not necessarily to become a coding style or restriction. Sounds fair to me.